CHANGES.md

```diff
@@ -1,4 +1,189 @@
+## stable-2.3.0
+
+### Announcing Linkerd 2.3 🎈
+
+This stable release introduces a new TLS Identity system into the default
+Linkerd installation, replacing `tls=optional` and the `linkerd-ca` controller.
```

Comment:
I recommending using `--tls=optional` to indicate that it's a flag. That's how flags are handled in the individual bullet points below.